components/nimbus/examples/README.md

```diff
@@ -0,0 +1,31 @@
+# Experiments CLI
+In this directory we have a command line interface that helps us interact with the Nimbus SDK.
```

Comment:
For [reasons](https://github.com/mozilla/application-services/blob/main/docs/design/test-faster.md), we prefer to keep example crates in the top-level `./examples` subdirectory rather than inside the crate directory. Not necessary for this PR, but could you please file a follow-up issue to move this example code out into the main `./examples` directory?

Reply:
SDK-235 has been filed.